code/modules/mob/living/basic/guardian/guardian_verbs.dm

```diff
@@ -0,0 +1,185 @@
+/// Pop out into the realm of the living.
+/mob/living/basic/guardian/proc/manifest(forced)
+	if (is_deployed() || isnull(summoner?.loc) || istype(summoner?.loc, /obj/effect) || (!COOLDOWN_FINISHED(src, manifest_cooldown) && !forced) || locked)
+		return FALSE
+	forceMove(summoner.loc)
```

Comment:
should we just have an explicit `isnull(summoner)` check here? It's kinda silly to have two `summoner?.var` accesses if right after the `if()` chain we just assume that the summoner exists for the `forceMove()` call